Lesson1/2.py

```diff
@@ -1,2 +1,3 @@
+# Вывести питон 1000 раз
```

Comment:
Directories should be lowercase.
Python files should be lowercase and should not begin with numbers.